### PR TITLE
First wave of backend refactoring. Add exceptions catch, reworked some methods.

### PR DESCRIPTION
**Pull request by AstralRomance**
Please, review and suggest your solutions for places which you think made bad.

### CODE REVIEW

File: src/databaseSvc/databaseManipulation.py [hunk 1]
```diff
@@ -60,19 +60,16 @@ def replace_event(self, event_id: str, new_event: dict) -> bool:
 
     @Logger()
     def delete_event(self, event_id: str) -> bool:
-        return self.session.delete_one({'Event_id': event_id}).deleted_count == 1
+        return self.session.delete_one({'Event_id': event_id}).deleted_count
```
Comment (NikitaCartes) [inline src/databaseSvc/databaseManipulation.py:63]: We expect bool as a result but `deleted_count` is int. Also what if it somehow delete more than one event?

File: src/databaseSvc/databaseManipulation.py [hunk 2]
```diff
@@ -60,19 +60,16 @@ def replace_event(self, event_id: str, new_event: dict) -> bool:
 
     @Logger()
     def delete_event(self, event_id: str) -> bool:
-        return self.session.delete_one({'Event_id': event_id}).deleted_count == 1
+        return self.session.delete_one({'Event_id': event_id}).deleted_count
 
-    #@Logger()
+    @Logger()
```
Comment (NikitaCartes) [inline src/databaseSvc/databaseManipulation.py:65]: We totally need to revisit why logger exist and do we need it.

File: src/databaseSvc/settings.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     server_host: str = '0.0.0.0'
-    server_port: int = DEPLOY_CONNECTION_PORT002
+    server_port: int = 8080
```
Comment (NikitaCartes) [inline src/databaseSvc/settings.py:6]: With that we will broke dev/main environment. We need to rewrite deploy.sh

File: src/eventManagerSvc/service/eventManagerSvc.py [hunk 4]
```diff
@@ -103,48 +178,56 @@ def update_player_points(self, event_id: str, player_id: str, round_number: int,
...
-    def generate_round(self, event_id: str, round_number: int):
+    # INFO: It looks better for now, but still suspective
+    def generate_round(self, event_id: str, round_number: int) -> RoundGenData:
         target_event = self.session.find_event(event_id)
         target_players_data = [{'name': player['Player_name'],
                                 'id': player['Player_id'],
-                                'Hidden_points': player['Hidden_points']} for player in target_event['Players'] if not player['Status']]
+                                'Hidden_points': player['Hidden_points']} for
+                                player in target_event['Players'] if not player['Status']]
+        random.seed(33)
```
Comment (NikitaCartes) [inline src/eventManagerSvc/service/eventManagerSvc.py:200]: Why not 42?

File: src/eventManagerSvc/settings.py [hunk 5]
```diff
@@ -3,12 +3,12 @@
...
     server_host: str = '0.0.0.0'
-    server_port: int = DEPLOY_CONNECTION_PORT000
+    server_port: int = 8080
     database_url: str
     player_svc_host = '0.0.0.0'
-    player_svc_port = DEPLOY_CONNECTION_PORT001
+    player_svc_port = 8080
     event_svc_host = '0.0.0.0'
-    event_svc_port = DEPLOY_CONNECTION_PORT002
+    event_svc_port = 8080
```
Comment (NikitaCartes) [inline src/eventManagerSvc/settings.py:11]: It use same port for all three services and that can't be right

**Review by NikitaCartes — CHANGES_REQUESTED** [review]

File: src/eventManagerSvc/settings.py (hunk 5 above)
Comment (AstralRomance) [reply src/eventManagerSvc/settings.py:11]: Yes, I just use same port for local debug. We need to rework our deployment scripts and get host/port from environment variables or something like this if we wouldn't use docker.

File: src/eventManagerSvc/service/eventManagerSvc.py (hunk 4 above)
Comment (AstralRomance) [reply src/eventManagerSvc/service/eventManagerSvc.py:200]: prost)

File: src/databaseSvc/settings.py (hunk 3 above)
Comment (AstralRomance) [reply src/databaseSvc/settings.py:6]: Yes, I think deployment things in your scope)0)

File: src/databaseSvc/databaseManipulation.py (hunk 1 above)
Comment (AstralRomance) [reply src/databaseSvc/databaseManipulation.py:63]: Right. Do we have any ways to check, what exactly thing was deleted from database (from mongo cursor or smth, not force compairing). We can return this from method.

**Pull request merged**